src/internal/tester/suite.go

```diff
@@ -0,0 +1,43 @@
+package tester
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type Suite interface {
+	suite.TestingSuite
+	Run(name string, subtest func()) bool
+}
+
+func NewUnitSuite(t *testing.T, envSets [][]string) *unitSuite {
```

Comment:
should unit tests ever depend on env vars?  Seems like an anti-pattern to me.

Reply:
excellent point. It did feel a bit odd to have there. Easy enough to remove though